src/Elewant/Webapp/Infrastructure/Doctrine/BreedMappingType.php

```diff
@@ -74,7 +77,9 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform): ?str
     }
 
     /**
-     * @inheritdoc
+     * @param AbstractPlatform $platform
+     * @return bool
+     * @phpcsSuppress SlevomatCodingStandard.Functions.UnusedParameter.UnusedParameter
```

Comment:
Yes, these suppressions can be done in the `phpcs.xml` file itself. For example:

```
<rule ref="PSR1.Methods.CamelCapsMethodName">
    <exclude-pattern>/(specs|tests)/</exclude-pattern>
</rule>
```

On one hand, I too dislike `@phpcsSuppress` (and `@noinspection`) in the code, but I dislike having to maintain them elsewhere even more.
Mainly because these suppressions tend to be added, but never removed. In my experience, nobody actually bothers to look at them and check if they're still needed.
So I prefer having them in the code where they tend to be maintained, over having them in `phpcs.xml` where they are forgotten.

Reply:
Sure.. I think it's likely to rot either way, but maybe less so when in the code. All it does is constantly remind me we're "improving" code in a bureaucratic way, rubber-stamping.